lld/include/lld-c/LLDAsLibraryC.h

```diff
@@ -131,10 +131,22 @@ void LLVMGetUndefinedReferencesEraVM(LLVMMemoryBufferRef inBuffer,
                                      char ***factoryDepSymbols,
                                      uint64_t *numFactoryDepSymbols);
 
-/** Disposes an array with linker symbols returned by the
+/** Disposes an array with symbols returned by the
  *  LLVMGetUndefinedReferencesEraVM(). */
-void LLVMDisposeUndefinedReferencesEraVM(char *linkerSymbolNames[],
-                                         uint64_t numLinkerSymbols);
+void LLVMDisposeUndefinedReferencesEraVM(char *symbolNames[],
```

Comment:
Any reason why we need two of these?

Reply:
I just didn't want to change the EraVM API here. OK, let's use a common endpoint - `LLVMDisposeUndefinedReferences`.